docker/grouper/Dockerfile

```diff
@@ -15,10 +15,11 @@ ENV ANT_URL $ANT_BASE/apache-ant-$ANT_VERSION-bin.tar.gz
 
 # Environment variables for tomcat.
 ENV CATALINA_HOME /opt/tomcat
+ENV TOMCAT_CONF /etc/tomcat
 ENV PATH $CATALINA_HOME/bin:$PATH
 ENV TOMCAT_MAJOR 7
-ENV TOMCAT_VERSION 7.0.68
-ENV TOMCAT_BASE https://www.apache.org/dist/tomcat
+ENV TOMCAT_VERSION 7.0.65
+ENV TOMCAT_BASE https://archive.apache.org/dist/tomcat
```

Comment:
This was done to work around an apparent conflict between the CSRF detection used by Grouper itself and the new `RestCsrfPreventionFilter` included in Tomcat 7.0.66.